wagtail_newsletter/campaign_backends/mailchimp.py

```diff
@@ -237,6 +253,37 @@ def send_campaign(self, campaign_id: str) -> None:
                 error, "Error while sending campaign", campaign_id=campaign_id
             )
 
+    def schedule_campaign(self, campaign_id: str, schedule_time: datetime) -> None:
+        rounded_minute = schedule_time.minute - (schedule_time.minute % 15)
+        rounded_time = schedule_time.replace(
+            minute=rounded_minute, second=0, microsecond=0
+        )
+        if rounded_time != schedule_time:
+            raise CampaignBackendError(
+                "Schedule time must be in 15 minute intervals,"
+                f" e.g. {schedule_time.hour}:{rounded_minute} "
+                f"not {schedule_time.hour}:{schedule_time.minute}."
```

Comment:
Sorry, just noticed that we need to format this in two digits for each hour and minute...

<img width="124" alt="image" src="https://github.com/user-attachments/assets/d36d748e-690f-4bdd-babe-b336428e1162">